### PR TITLE
SI also uses OriginatingEndpoint and ProcessingEndpoint headers

### PR DESCRIPTION
**Pull request by ramonsmits**
Added `OriginatingEndpoint` and `ProcessingEndpoint` to comment which headers are used by SI to link messages.

### CODE REVIEW

File: nservicebus/messaging/headers.md [hunk 1]
```diff
@@ -306,6 +306,7 @@ The timestamp when the message was sent. Used by the [Performance Counters](/mon
 
 The endpoint name the message was sent from.
 
+INFO: Used for linking messages in ServiceInsight. See [NServiceBus.ConversationId](#nservicebus.conversationid)
```
Comment (WilliamBZA) [inline nservicebus/messaging/headers.md:309]:
```suggestion
INFO: Used for linking messages in ServiceInsight. See [NServiceBus.ConversationId](#nservicebus-conversationid)
```

File: nservicebus/messaging/headers.md [hunk 2]
```diff
@@ -331,12 +332,12 @@ The timestamp when the processing of a message ended.
 
 Name of the endpoint where the message was processed.
 
+INFO: Used for linking messages in ServiceInsight. See [NServiceBus.ConversationId](#nservicebus.conversationid)
```
Comment (WilliamBZA) [inline nservicebus/messaging/headers.md:335]:
```suggestion
INFO: Used for linking messages in ServiceInsight. See [NServiceBus.ConversationId](#nservicebus-conversationid)
```

**Review by WilliamBZA — APPROVED** [review]
Just need to fix the anchor links

File: nservicebus/messaging/headers.md [hunk 3]
```diff
@@ -306,6 +306,7 @@ The timestamp when the message was sent. Used by the [Performance Counters](/mon
 
 The endpoint name the message was sent from.
 
+INFO: Used for linking messages in ServiceInsight. See [NServiceBus.ConversationId](#nservicebus-conversationid)
```
Comment (ramonsmits) [inline nservicebus/messaging/headers.md:309]:
```suggestion
INFO: Used for linking messages in ServiceInsight. See [NServiceBus.ConversationId](#nservicebusconversationid)
```

File: nservicebus/messaging/headers.md [hunk 4]
```diff
@@ -331,12 +332,12 @@ The timestamp when the processing of a message ended.
 
 Name of the endpoint where the message was processed.
 
+INFO: Used for linking messages in ServiceInsight. See [NServiceBus.ConversationId](#nservicebus-conversationid)
```
Comment (ramonsmits) [inline nservicebus/messaging/headers.md:335]:
```suggestion
INFO: Used for linking messages in ServiceInsight. See [NServiceBus.ConversationId](#nservicebusconversationid)
```

File: nservicebus/messaging/headers.md [hunk 5]
```diff
@@ -306,6 +306,7 @@ The timestamp when the message was sent. Used by the [Performance Counters](/mon
 
 The endpoint name the message was sent from.
 
+INFO: Used for linking messages in ServiceInsight. See [NServiceBus.ConversationId](#nservicebusconversationid)
```
Comment (ramonsmits) [inline nservicebus/messaging/headers.md:309]:
```suggestion
INFO: Used for linking messages in ServiceInsight. See [NServiceBus.ConversationId](#messaging-interaction-headers-nservicebus-conversationid)
```

File: nservicebus/messaging/headers.md [hunk 6]
```diff
@@ -331,12 +332,12 @@ The timestamp when the processing of a message ended.
 
 Name of the endpoint where the message was processed.
 
+INFO: Used for linking messages in ServiceInsight. See [NServiceBus.ConversationId](#nservicebusconversationid)
```
Comment (ramonsmits) [inline nservicebus/messaging/headers.md:335]:
```suggestion
INFO: Used for linking messages in ServiceInsight. See [NServiceBus.ConversationId](#messaging-interaction-headers-nservicebus-conversationid)
```

**Pull request merged**